### PR TITLE
III-4428 Add endpoint to delete image from organizer

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added `ImageRemoved` event
- Added `RemoveImage` command
- Implement `removeImage` on `Organizer` aggregate
- Implement `ImageRemoved` inside `OrganizerLDProjector`
- Added `RemoveImageHandler` to handle the `RemoveImage` command
- Added `DeleteImageRequestHandler` to handle the `DELETE` image request

---
Ticket: https://jira.uitdatabank.be/browse/III-4428

### CODE REVIEW

File: tests/Organizer/OrganizerTest.php [hunk 1]
```diff
@@ -741,6 +741,71 @@ function (Organizer $organizer) {
         ];
     }
 
+    /**
+     * @test
+     * @dataProvider addImageDataProvider
```
Comment (jdrieghe) [inline tests/Organizer/OrganizerTest.php:746]: Is this using the wrong data provider?

**Review by jdrieghe — CHANGES_REQUESTED** [review]
One of the tests is using an incorrect data provider

File: tests/Organizer/OrganizerTest.php (hunk 1 above)
Comment (jdrieghe) [reply tests/Organizer/OrganizerTest.php:746]:
```suggestion
     * @dataProvider removeImageDataProvider
```

Comment (LucWollants) [reply tests/Organizer/OrganizerTest.php]: Nicely spotted!

**Review by jdrieghe — APPROVED** [review]

**Pull request merged**